src/components/entries/TextField.js

```diff
@@ -53,10 +56,10 @@ function Textfield(props) {
         autoComplete="off"
         disabled={ disabled }
         class="bio-properties-panel-input"
-        onInput={ handleInput }
-        onFocus={ props.onFocus }
-        onBlur={ props.onBlur }
```

Comment:
Good that we removed these props; we never used them (in the entry components). 👍